docs/_layouts/blogpost.html

```diff
@@ -0,0 +1,73 @@
+---
+layout: default
+section_id: blog
+---
+{% assign num_posts = site.posts | size %}
+
+<!-- Remove the top padding but add in bottom padding -->
+<div class='full white' style="padding:0; padding-bottom:10px;">
+  <!-- Blog entries sit in a single row -->
+  <div class='row'>
+    <!-- Create the Title of the page -->
+    <div class="special-title centered-text">
+      <h2>The Latest in Open GEE</h2>
+      <!-- Add in a small seperator -->
+      <p class="shortline"/>
+    </div>
+    <!--  The large div contains the width of the entire page -->
+    <div class='large-12'>
+      <!-- The post content resides in the left 75% of the page -->
+      <div class='large-9 columns'>
+        <!-- Add in the Post Header content -->
+        <div class='post'>
+          <!-- Create a nice timestamp icon -->
+          <div class='timestamp left text-center'>
+            <span class='month'>{{ page.date | date: "%h" }}</span>
+            <span class='date'>{{ page.date | date: "%d" }}</span>
+            <!--  Reuse the month formatting to add the year to the post-->
+            <span class='month'>{{ page.date | date: "%Y" }}</span>
+          </div>
+          <!-- Adds the title of the page, thumbnail and author -->
+          <div class='info'>
+            <h3 class='regularText'>
+              {{ page.title }}
+            </h3>
+            {% if page.thumbnail %}
+              <img src="{{site.github.url}}/images/{{page.thumbnail}}" height="75" width="75">
+            {% endif %}
+            <p>
+              By
+              <span class='author'>{{ page.author }}</span>
+            </p>
+          </div>
+        </div>
+        <!-- This is the actual content of the post -->
+        <div class="content">
+          {{ content }}
+        </div>
+
+      </div>
+      <!-- Only display the Previous Posts section if there are >1 posts -->
+      {% if num_posts > 1 %}
+        <!-- The previous posts navigator sits in the right 25% -->
+        <div class='large-3 columns large-uncentered opposite darkBorder'>
+          <div>
+            <h3 class='regularText'> Previous Posts </h3>
```

Comment:
These aren't all going to be Previous Posts. So we should come up with a better name, but I don't really know what would be a good name. "All Posts"? "More Posts"?